packages/ui-core/src/typography/text.component.tsx

```diff
@@ -1,11 +1,12 @@
-import React from 'react';
+import React, { FC, forwardRef } from 'react';
 import styled, { css } from 'styled-components';
 import { typography } from 'styled-system';
 import { Typography } from '../types';
 
 type Props = {
   children: React.ReactNode;
   truncate?: boolean;
+  ref?: any;
```

Comment:
IMO better type for ref is: `HTMLDivElement | null`

Reply:
it's outdated